jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/install-common/install-common.sh

```diff
@@ -120,3 +120,19 @@ function configure_drivers(){
   )
 }
 
+function run_cli_script(){
+  if [ $# != 1 ]; then
+    echo "Usage: Script file parameter required."
```

Comment:
Should we use log_error? I am wandering if we should exit 1? I see that other functions just print the message and return. We are possibly hiding a user mis-configuration that could then be difficult to identify at runtime.

Reply:
I asked myself the same question, but the other methods exposed in the same file use also a simple echo. So finally, I wanted to be consistent.

I'm not sure if we can change all, just only the new one. We could use a warning instead to make the trace more relevant.

WDYT?

Reply:
@yersan , I would not touche the existing one. This is perhaps just something we would need to document for the API you introduce. You could add a chapter in WildFly s2i README.md